pronto/static/js/entry/annotations.js

```diff
@@ -151,9 +148,9 @@ export function refresh(accession) {
                         } else
                             i++;
 
-                        text = text.replace(arr[0], `<a data-ref href="#${pubID}">${i}</a>`);
+                        return `<a data-ref href="#${pubID}">${i}</a>`
                     }
```

Comment:
Return original match if `pubID` not found.

```suggestion
                    }
                    
                    return match;                    
```